src/stack_pr/cli.py

```diff
@@ -1313,6 +1313,13 @@ def create_argparser(
         ),
         help="List of reviewers for the PR",
     )
+    parser_submit.add_argument(
+        "-s",
+        "--stash",
+        action="store_true",
+        default=config.getboolean("common", "stash", fallback=False),
```

Comment:
Could you please add a mention of this to `README.md` ("Config files" section)?

Reply:
done